CovertInfiltration/Src/CovertInfiltration/Classes/X2SitRep_InfiltrationSitRepEffects.uc

```diff
@@ -0,0 +1,594 @@
+	Templates.AddItem(Manager.FindAbilityTemplate('PlaceEvacZone'));
+	Templates.AddItem(Manager.FindAbilityTemplate('LiftOffAvenger'));
+
+	Templates.AddItem(Manager.FindAbilityTemplate('Loot'));
+	Templates.AddItem(Manager.FindAbilityTemplate('CarryUnit'));
+	Templates.AddItem(Manager.FindAbilityTemplate('PutDownUnit'));
+
+	Templates.AddItem(Manager.FindAbilityTemplate('Interact_PlantBomb'));
+	Templates.AddItem(Manager.FindAbilityTemplate('Interact_TakeVial'));
+	Templates.AddItem(Manager.FindAbilityTemplate('Interact_StasisTube'));
+	Templates.AddItem(Manager.FindAbilityTemplate('Interact_MarkSupplyCrate'));
+	Templates.AddItem(Manager.FindAbilityTemplate('Interact_ActivateAscensionGate'));
+
+	Templates.AddItem(Manager.FindAbilityTemplate('DisableConsumeAllPoints'));
+
+	Templates.AddItem(Manager.FindAbilityTemplate('Revive'));
+	Templates.AddItem(Manager.FindAbilityTemplate('Panicked'));
+	Templates.AddItem(Manager.FindAbilityTemplate('Berserk'));
+	Templates.AddItem(Manager.FindAbilityTemplate('Obsessed'));
+	Templates.AddItem(Manager.FindAbilityTemplate('Shattered'));
+
+    return Templates;
+}
+
+static function X2SitRepEffectTemplate CreateTacticalAnalysisEffectTemplate_CI()
+{
+    local X2SitRepEffect_ModifyTacticalStartState Template;
+
+    `CREATE_X2TEMPLATE(class'X2SitRepEffect_ModifyTacticalStartState', Template, 'TacticalAnalysisEffect_CI');
+
+    Template.ModifyTacticalStartStateFn = TacticalAnalysisStartModifier;
+
+    return Template;
+}
+
+static function TacticalAnalysisStartModifier(XComGameState StartState)
+{
+    local XComGameState_Player Player;
+    local Object PlayerObject;
+
+    foreach StartState.IterateByClassType(class'XComGameState_Player', Player)
+    {
+        if (Player.GetTeam() == eTeam_XCom)
+        {
+            break;
+        }
+    }
+
+    PlayerObject = Player;
+
+    `XEVENTMGR.RegisterForEvent(PlayerObject, 'ScamperEnd', Player.TacticalAnalysisScamperResponse, ELD_OnStateSubmitted);
+}
+
+// best to use this in conjunction
+static function X2SitRepEffectTemplate CreateAdventAirPatrolsEffectTemplate_CI()
+{
+    local X2SitRepEffect_ModifyTacticalStartState Template;
+
+    `CREATE_X2TEMPLATE(class'X2SitRepEffect_ModifyTacticalStartState', Template, 'AdventAirPatrolsEffect_CI');
+
+    Template.ModifyTacticalStartStateFn = AdventAirPatrolStartModifier;
+
+    return Template;
+}
+
+static function AdventAirPatrolStartModifier(XComGameState StartState)
+{
+    local XComGameState_Player Player;
+    local Object PlayerObject;
+
+    foreach StartState.IterateByClassType(class'XComGameState_Player', Player)
+    {
+        if (Player.GetTeam() == eTeam_XCom)
+        {
+            break;
+        }
+    }
+
+    PlayerObject = Player;
+
+    `XEVENTMGR.RegisterForEvent(PlayerObject, 'SquadConcealmentBroken', CallReinforcements, ELD_OnStateSubmitted);
+    `XEVENTMGR.RegisterForEvent(PlayerObject, 'SpawnReinforcementsComplete', CallReinforcements, ELD_OnStateSubmitted);
```

Comment:
This is not how this works -.- If you put `PlayerObject` as first argument, then the callback should be on `PlayerObject` as well. If it's not - move it to `X2EventListener_Infiltration` and simply check if sitrep is active before doing anything